pipeline/dbt/models/intermediate/sources/soliguide/int_soliguide__structures.sql

```diff
@@ -22,7 +22,15 @@ final AS (
         NULL                                                          AS "typologie",
         lieux.updated_at                                              AS "date_maj",
         NULL                                                          AS "siret",
-        RTRIM(SUBSTRING(lieux.name, 1, 150), '.')                     AS "nom",
+        REGEXP_REPLACE(
+            REGEXP_REPLACE(
+                RTRIM(SUBSTRING(lieux.name, 1, 150)),
```

Comment:
```suggestion
                CASE WHEN LENGTH(lieux.name) <= 150 THEN lieux.name ELSE LEFT(lieux.name, 149) || '…' END
```

tronquer avec une ellipse